challengeMk2/DataBase/ILocalDataService.cs

```diff
@@ -1,20 +1,21 @@
 using System;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 using ChallengeMk2.Models;
 
 namespace ChallengeMk2.DataBase
 {
     public interface ILocalDataService
     {
-        void Initialize();
+        Task Initialize();

```

Comment:
General remark, async methods should be named with an `Async' suffix.
See https://docs.microsoft.com/en-us/dotnet/csharp/async#important-info-and-advice

Reply:
yep, i have to change all of async method names all around the solution.

Reply:
Should i take solution files one by one to rename async method properly or change names only when i see it within the class i am currently working on ?